### PR TITLE
workload: handle missing removed sys col in RESTORE results

### PR DESCRIPTION
**Pull request by dt**
This was removed in favor of just reporting row counts however workload was
expecting it to be there (and discarding it) and needs to handle it still
being there when run against older versions.

Release note: none.

### CODE REVIEW

File: pkg/ccl/workloadccl/fixture.go [hunk 1]
```diff
@@ -417,12 +417,34 @@ func importFixtureTable(
...
+		return 0, err
+	}
+	defer res.Close()
+	resCols, err := res.Columns()
 	if err != nil {
 		return 0, err
 	}
+	if len(resCols) == 7 {
+		if !res.Next() {
+			return 0, gosql.ErrNoRows
```
Comment (tbg) [inline pkg/ccl/workloadccl/fixture.go]: This can be lifted up.

File: pkg/ccl/workloadccl/fixture.go [hunk 2]
```diff
@@ -417,12 +417,34 @@ func importFixtureTable(
 	}
 	var rows, index, tableBytes int64
 	var discard driver.Value
-	err := sqlDB.QueryRow(buf.String(), params...).Scan(
-		&discard, &discard, &discard, &rows, &index, &tableBytes,
-	)
+	res, err := sqlDB.Query(buf.String(), params...)
```
Comment (tbg) [inline pkg/ccl/workloadccl/fixture.go]: Would it be better to name the columns and avoid the branching below? Of course only if the column names never changed.

Comment (tbg) [reply pkg/ccl/workloadccl/fixture.go]: (`SELECT foo, bar, baz FROM [ IMPORT TABLE ... ]`)

**Review by tbg — APPROVED** [review]
Assume you took this out for a spin already.

File: pkg/ccl/workloadccl/fixture.go (hunk 2 above)
Comment (dt) [reply pkg/ccl/workloadccl/fixture.go]: heh, I tied that first actually -- doesn't work since the optimizer doesn't know about import/restore apparently.

**Pull request merged**